src/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const FORTA_BOT_REGISTRY: string = "0x61447385B019187daa48e91c55c02AF1F1f3F863";
+export const BOT_DEPLOYER_ADDRESS: string = "0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8";
+export const NEW_AGENT_FUNCTION_SIGNATURE: string =
+  "function newAgent(uint256 agentId, address owner, string metadata, uint256[] chainIds) public";
+export const INCORRECT_FUNCTION_SIGNATURE: string =
+  "function fooAgent(uint256 agentId, address owner, string metadata, uint256[] chainIds) public";
+export const EXTERNAL_AGENT_FUNCTION_SIGNATURE: string =
+  "function newAgent(uint256 agentId, address owner, string metadata, uint256[] chainIds) external";
```

Comment:
See comment in the test file regarding the last test, where this variable is used.